strr-platform-web/tests/e2e/test-utils/auth-setup.ts

```diff
@@ -48,11 +47,11 @@ async function authSetup () {
     throw new Error('User or password is not set in environment variables.')
   }
 
-  await page.goto(baseURL + 'en-CA/platform/application/')
-  await expect(page.getByTestId('h1')).toContainText(enI18n.global.t('platform.title.application'))
-  await page.locator('[id="logged-out-options-dropdown"]').click()
-  await expect(page.getByRole('menuitem', { name: 'IDIR' })).toBeVisible() // will need to change this to BCeID
-  await page.getByRole('menuitem', { name: 'IDIR' }).click()
+  // replace with environment variable?
+  const signinUrl = 'https://dev.account.bcregistry.gov.bc.ca/signin/bceid/'
```

Comment:
yeah I think we could do ${...NUXT_AUTH_WEB_URL}/signin/bceid

Reply:
I'm just thinking if this is actually better than what I had previously or not. 

Pros: 
- handles login for dev/test/prod without concerning the test runner with any middleware changes etc we make
- less steps, marginally faster 

Cons:
- we're relying on auth-web to never change this url redirect

I doubt that will change anytime soon but 🤷‍♂️